pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -30,7 +30,8 @@ let
 
     hardeningDisable = lib.optional stdenvNoCC.buildPlatform.isDarwin "format";
 
-    sourceRoot = lib.optionalString stdenvNoCC.hostPlatform.isAndroid ".";
+    # FIXME: remove, was left over to prevent a mass rebuild
+    sourceRoot = "";
```

Comment:
This is already targeting staging, so this will not prevent mass rebuilds for consumers.

Reply:
Fair enough, was mainly useful for testing this change. Removed in force push.